source/views/home/index.tsx

```diff
@@ -91,7 +91,7 @@ export const NavigationKey = 'Home'
 export const NavigationOptions: NativeStackNavigationOptions = {
 	title: 'All About Olaf',
 	headerBackTitle: 'Home',
-	headerRight: (props) => <OpenSettingsButton canGoBack={true} {...props} />,
+	headerRight: (props) => <OpenSettingsButton {...props} />,
```

Comment:
Shouldn't we put canGoBack on the right side, to ensure that it's always true?

Reply:
I think it is already a provided prop on the right side so this would fix a warning about it being added twice. Unless I'm not understanding you correctly and you're saying to replace the value some other way?

Reply:
Because `props.canGoBack` is a required property, it will always be set, so it will always overwrite our `canGoBack={true}` attribute.

I think I'm proposing that we set it on the right side, after `{…props}` is expanded, so that we ignore the value from `props` and always set it to be `true`.

I don't remember why we put `canGoBack=true` in the first place, but I imagine that somehow we could configure it to be false in `props`… is that really a problem, though?

```tsx
# Current:
	headerRight: (props) => <OpenSettingsButton canGoBack={true} {...props} />,
# Proposed:
	headerRight: (props) => <OpenSettingsButton {...props} />,
# Alternate Proposal:
	headerRight: (props) => <OpenSettingsButton {...props} canGoBack={true} />,
```

Reply:
Okay. I've convinced myself - let's just do what you have here.

Reply:
Alright. If we find a need for it we can come back to this!